docs/installation/docker.md

```diff
@@ -0,0 +1,120 @@
+
+## Installing Docker
+
+If you are installing Docker, we recommend using the application [Docker Desktop](https://www.docker.com/products/docker-desktop). It provides a GUI for managing Docker container in Windows and MacOS, along with the Docker engine and suite of command-line tools. Linux users can download the Engine and command-line tools from that same link.
+
+There is also a legacy project called [Docker Toolbox](https://docs.docker.com/toolbox/overview/) which may be of interest if your machine cannot run Docker Desktop, or if you already have it installed.
+
+
+[Download Docker](https://www.docker.com/products/docker-desktop)
+
+
+## Launching Islandora with Docker
+
+In a shell, clone the isle-dc project. In that directory, enter the command `make` (or `make dev` - see note) to build the docker infrastructure. Then use the command `docker-compose up -d` to start the containers.
+
+!!! Note
+    The `make` command alone will spin up a sandbox-like version of Isle on the front end, but the code files will be inaccessible. The `make dev` command will also copy the active code files locally in a way that they are live to the Isle site. This method takes longer, but is required if you will be testing pull requests or writing code.
+
+```bash
+git clone https://github.com/islandora-devops/isle-dc
+cd isle-dc
+make
+```
+
+
+Results:
+```
+isle-dc$ make
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  3562  100  3562    0     0  12779      0 --:--:-- --:--:-- --:--:-- 12812
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  1708  100  1708    0     0   6809      0 --:--:-- --:--:-- --:--:--  6832
+docker-compose pull
+Pulling activemq   ... done
+Pulling alpaca     ... done
+Pulling blazegraph ... done
+Pulling cantaloupe ... done
+Pulling fcrepo     ... done
+Pulling fits       ... done
+Pulling crayfits   ... done
+Pulling gemini     ... done
+Pulling homarus    ... done
+Pulling houdini    ... done
+Pulling hypercube  ... done
+Pulling mariadb    ... done
+Pulling matomo     ... done
+Pulling milliner   ... done
+Pulling recast     ... done
+Pulling solr       ... done
+Pulling drupal     ... done
+Pulling traefik    ... done
+Pulling watchtower ... done
+```
+
+!!! Troubleshooting
+    If you get an error such as: `ERROR: Version in "./docker-compose.activemq.yml" is unsupported.`, then you need to upgrade Docker, and then enter the command `make clean`. If you forget to `make clean`, then the next time `make` runs you may see an error such as: `ERROR: Top level object in './docker-compose.yml' needs to be an object not '<class 'NoneType'>'.`
+
+!!! Troubleshooting
+    If you get an error such as: `make[1]: *** [create-codebase-from-demo] Error 143 make: *** [dev] Error 21` ... documentation to come.
```

Comment:
I just hit this....

Reply:
Y'all are right. Macs need more time so I made a PR. https://github.com/Islandora-Devops/isle-dc/pull/114

Tested on my local mac.

Reply:
Fixed with that PR; I removed the section talking about `make[1]: *** [create-codebase-from-demo] Error 143 make: *** [dev] Error 21`  - hopefully people dont' get it again, and/or find it in this issue thread.